docs/kubernetes/AWSIntegration.md

```diff
@@ -37,15 +37,15 @@ sudo mv calico-etcd.manifest /etc/kubernetes/manifests/
 
 After a short delay, the kubelet on your master will automatically create a container for the new etcd which can be accessed on port 6666 of your master.
 
-Next, use `calicoctl` to spin up the `calico/node` container and install the [networking plugin](https://github.com/projectcalico/calico-kubernetes) 
+Next, use `calicoctl` to spin up the `calico/node` container and install the Calico [network plugin](https://github.com/projectcalico/calico-kubernetes) for Kubernetes. 
 ```
-sudo ETCD_AUTHORITY=<MASTER_PRIVATE_IPV4>:6666 calicoctl node --kubernetes --kube-plugin-version=v0.4.0
+sudo ETCD_AUTHORITY=<MASTER_IPV4>:6666 calicoctl node --kubernetes --kube-plugin-version=v0.4.0
```

Comment:
AWS hosts have a public and a private IP address. I believe you need to use the Private address, although it may not matter.